### ISSUE TITLE
[Crash]: FTB Quests crashing server on crafting upgrade for sophisticated backpacks

### ISSUE THREAD

**Issue by violetkitty87** [AllTheMods/ATM-9#1526]
### Possible Fixes

No

### Modpack Version

0.2.54

### What happened?

[crash-2024-03-22_15.51.16-server.txt](https://github.com/AllTheMods/ATM-9/files/14729129/crash-2024-03-22_15.51.16-server.txt)
this is a consistant repeatable crash as every time i go to make the item it crashes

### Crash Log

<script src="https://gist.github.com/violetkitty87/78e343a658b4a036cf51abbf4ef25c07.js"></script>

**Comment by Baylem** [comment]
This is likely related to #1505

**Comment by violetkitty87** [comment]
that is likely but i did add on the crash report to help with diagnosing

**Comment by violetkitty87** [comment]
also the version on that one you linked is an older version im on latest

**Comment by Baylem** [comment]
@violetkitty87 Yeah, this is a very similar crash report and this issue is affecting every version after .51
